controllers/members.js

```diff
@@ -139,10 +143,48 @@ const archiveMembers = async (req, res) => {
   }
 };
 
+/**
+ * Purges the Cache of Members Profile Page
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const purgeMembersCache = async (req, res) => {
+  try {
+    const { username } = req.params;
```

Comment:
We should get the username to construct the cache clearing URL from the cookie, not from the params. (Otherwise we could potentially clear the cache for someone else 🤷🏻‍♂️)

Reply:
Made the change